src/ServiceInsight/Startup/AppBootstrapper.cs

```diff
@@ -27,6 +28,8 @@ public AppBootstrapper()
 
         protected override void Configure()
         {
+            ApplicationConfiguration.Initialize();
```

Comment:
Would this be better placed in `OnStartup`?

The docs for `Configure` say:

> Override to configure the framework and setup your IoC container.

But this method neither configures the framework nor sets up the IoC container.

The docs for `OnStartup` are not very specific:

> Override this to add custom behavior to execute after the application starts.

However, `OnStartup` receives a `StartupEventArgs` which has an `Args` property, and the docs for that say:

> Gets command line arguments that were passed to the application from either the command prompt or the desktop

This suggests that `OnStartup` is the dual of `Program.Main` so I think it makes sense to read the app config here.

Reply:
I didn't want to put too many things into the app startup, since we need to:

1. Start logging, wire exception handler
2. Show the splash screen.

If we do this before step 1, it might blow up with an unhandled exception. If we keep adding stuff before step 2, the splash screen will take long to appear which beats its purpose.

In this case, I think since there's little to do, between step 1 and 2 should be okay. That means it will be moved to the end of the App constructor. Should I move it there?